webapp/src/ts/modules/tasks/tasks.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="filters">
+  <!--div class="inner">
+    <mm-navigation></mm-navigation>
+  </div-->
+</div>
+
+<div class="inner">
+  <div class="inbox page">
+
+    <div id="tasks-list" class="col-sm-4 inbox-items left-pane">
+      <div class="selection-count" *ngIf="selectMode">
```

Comment:
I think you've copied the component file from the reports module instead of migrating the task templates (https://github.com/medic/cht-core/blob/master/webapp/src/templates/partials/tasks.html). Can you, please, migrate the template and change the necessary code instead of copying the template from another module?